doc/js_tutorials/js_assets/adapter.js

```diff
@@ -0,0 +1,3971 @@
+(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.adapter = f()}})(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```

Comment:
What is the source/license of this file?

Reply:
It's BSD license: https://github.com/webrtc/adapter/blob/master/LICENSE.md. We can add the license header.

Or we can point to the online version in tutorials, e.g. https://webrtc.github.io/adapter/adapter-latest.js

Which option do you prefer to?

Reply:
Thanks!

Online version should be good, but please check that:
- tests via browser works
- tests via `node` works too

Also we probably should point to some [fixed version](https://webrtc.github.io/adapter/adapter-5.0.4.js) instead of "latest", they are available too: https://github.com/webrtc/adapter/tree/gh-pages

Reply:
Currently, tests don't depend on `adapter.js`. Only the tutorials depend on it. We will make sure all tutorials work with the online version of adapter.js.

I will use the latest stable version as fixed version.